### DISCUSSION TITLE
How to download external themes using new operator?

### DISCUSSION THREAD

**Discussion by sirockin** [keycloak/keycloak#47098] [Q&A]
### Description

The [legacy keycloak operator](https://github.com/keycloak/keycloak-operator) provides a means to specify a url to a `.jar` file to be downloaded and made availabe to running keycloak instances.

Is this same functionality available in the new operator? I don't see any reference to it in the [current docs](https://www.keycloak.org/guides#operator)

Thanks!

**Comment by TheKnarf** [comment]
Semi related to #13418

**Comment by shawkins** [comment]
That functionality is not part of the current operator. New relevant issues:

#44276
#45442
